openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -832,6 +833,50 @@ Here is another example JWT not only proving possession of a private key but als
 }
 ```
 
+#### `ldp_vp` Key Proof Type {#ldp_vp-proof-type}
+
+The verifiable presentation as defined by [@VC_DATA_2.0] or [@VC_DATA] signed using Data Integrity that are used as Key Proofs MUST contain the following elements:
+
+  * in the presentation itself,
```

Comment:
```suggestion
  * in the payload of a W3C Verifiable Presentation itself,
```
not sure `payload` is most appropriate, but something along these lines

Reply:
https://github.com/openid/OpenID4VCI/pull/66/commits/9e31825398e426fb22da4c09c7259e0647dcb677